packages/ag-charts-website/project.json

```diff
@@ -17,14 +17,9 @@
       }
     },
     "build-tsc": {
-      "executor": "@nx/js:tsc",
-      "inputs": ["tsDefaults"],
-      "outputs": ["{options.outputPath}"],
+      "executor": "nx:run-commands",
       "options": {
-        "main": "packages/ag-charts-website/src",
-        "outputPath": "dist/packages/ag-charts-website/out-tsc",
-        "tsConfig": "packages/ag-charts-website/tsconfig.json",
-        "compiler": "tsc"
+        "command": "tsc --project packages/ag-charts-website/tsconfig.json"
       }
```

Comment:
What was the motivation to switch to the `tsc` command-line tool? I'd like to try and minimise this scenario, as it adds more edge cases on upgrade vs. following the 'standard' Nx approach?

Reply:
I'm not sure what the nx/js:tsc is running, but it doesn't output any of the errors and passes while `tsc` fails correctly.